README.md

```diff
@@ -25,64 +25,10 @@ implementation 'com.grafana:grafana-opentelemetry-starter:1.2.0'
 <dependency>
     <groupId>com.grafana</groupId>
     <artifactId>grafana-opentelemetry-starter</artifactId>
-    <version>1.2.0</version>
+    <version>1.3.0</version>
 </dependency>
 ```
 
```

Comment:
The README now makes no mention about logging.  I think we should include a section to explain that OpenTelemetryAppender is auto configured.

Reply:
We have https://github.com/grafana/grafana-opentelemetry-starter/blame/d62278e62bf6169b27dfd521e2d31cff68f14ee9/README.md#L3

If we describe the OpenTelemetryAppender, we'd also have to describe other implementation details - but why?

Reply:
I think it's worthwhile noting that if logback or log4j is found/returned, then the starter will add the OpenTelemetryAppender.  But your call.